consensus/istanbul/core/core.go

```diff
@@ -227,11 +227,17 @@ func UnionOfSeals(aggregatedSignature types.IstanbulAggregatedSeal, seals Messag
 			panic(fmt.Sprintf("couldn't get address index for address %s", hex.EncodeToString(v.Address[:])))
 		}
 
+		var commit *istanbul.CommittedSubject
+		err = v.Decode(&commit)
+		if err != nil {
+			panic(fmt.Sprintf("couldn't decode committedSubject for address %s", hex.EncodeToString(v.Address[:])))
```

Comment:
Please return an error instead

Reply:
I modified `UnionOfSeals` to return an error instead of panicking.  I also modified `backend.Prepare` to handle that error.  PTAL.